lib/libc/minimal/include/sys/types.h

```diff
@@ -16,10 +16,11 @@ typedef unsigned int mode_t;
 #if !defined(__ssize_t_defined)
 #define __ssize_t_defined
 
-/* parasoft suppress item MISRAC2012-RULE_20_4-a item MISRAC2012-RULE_20_4-b
- * "Trick compiler to make sure the type of ssize_t won't be
- * unsigned long. View details in commit b889120"
+/* parasoft suppress item MISRAC2012-RULE_20_4-a
+ * "Trick compiler to make sure the type of ssize_t
+ * won't be unsigned long. View details in commit b889120"
  */
+/* parasoft suppress item MISRAC2012-RULE_20_4-b */
```

Comment:
I wonder if it would be better to add the `parasoft` comments separate from the regular "explanation" comments, e.g.:
```suggestion
/* parasoft suppress item MISRAC2012-RULE_20_4-a */
/* parasoft suppress item MISRAC2012-RULE_20_4-b */
/ * "Trick compiler to make sure the type of ssize_t
 * won't be unsigned long. View details in commit b889120"
 */
```

p.s. once again, these conventions should probably be documented somewhere.

Reply:
https://docs.parasoft.com/display/CPPTEST1031/Suppressing+Findings
It is official page, but it has lack of special cases. Unfortunately, I don't have direct access to the Parasoft tool to test how it detects all the tags. So I can see results only after Parasoft built zephyr project and published results..

Reply:
The idea to keep them together, is that in Parasoft tool you can see description in the table, without looking into code.
But, it is a good point to discuss. Should we add explanation comment separately or add it inside of the parasoft tag. If we will use Parasoft tool, definitely it is better to add comment inside of the tag.
If someday we will stop using parasoft tool, it is better to have explanation comments separately..

Reply:
@stephanosio I made changes, pls check if it looks better. Right now is a very experimental process, that we have to define together with the community. No any Zephyr Parasoft tags guidelines so I'm doing it according to my taste. Sure, in the future when more reviews and comments will come from reviewers, guidelines will be defined in natural way.